blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -68,6 +70,7 @@ export class PipelinePage extends Component {
         const isReady = !!pipeline;
 
         if (!pipeline && this.error) {
+            logger.log(`Error finding pipeline page for ${fullName}.`, this.error);
```

Comment:
Def not `error` and I don't think `warn` either ... it's not an "error" in the sense that the app is not working correctly. It's just a user doing something strange pasting in a wrong url or something.